packages/angular_devkit/build_angular/src/webpack/plugins/postcss-cli-resources.ts

```diff
@@ -155,7 +155,7 @@ export default function (options?: PostcssCliResourcesOptions): Plugin {
       }
 
       const value = decl.value;
-      const urlRegex = /url\(\s*(?:"([^"]+)"|'([^']+)'|(.+?))\s*\)/g;
+      const urlRegex = /url(?:\(['"]?)(.*?)(?:['"]?\))/g;
```

Comment:
It looks like this will match potentially invalid quote character combinations such as `url('abc.svg")` or `url("abc.svg)`.
Whitespace also appears to be valid but won't be captured here: `url ( "abc.svg" )`.

Reply:
Looks like the first too cases were not being handled before as well. Although such cases would cause the compilation to fail at different point.

Thoughts about changing it to: `url(?:\(\s*['"]?)(.*?)(?:['"]?\s*\))`?

Reply:
sounds good for now and fixes the linked issue